### ISSUE TITLE
Add custom headers to HTTP response

### ISSUE THREAD

**Issue by SBenter** [gambol99/keycloak-proxy#371]
I would like to have a feature for adding custom Headers to the HTTP Response towards the client (similar to `--headers`).

This way one could add new headers without a new keycloak-proxy deployment and set HPKP, HSTS, Expect-CT, Referrer-Policy etc.
Config example:
```yaml
response-headers:
  - Strict-Transport-Security: 'max-age=31536000'
  - Referrer-Policy: 'same-origin'
  - X-Permitted-Cross-Domains: 'none'
```
Thanks!

EDIT:
I am just digging through the code... It might be easier to expose the additional Headers by unrolled/secure. I think I will add a pull request for that in a few days.

**Comment by gambol99** [comment]
fixed in https://github.com/gambol99/keycloak-proxy/pull/386 and released in #388

**Comment by SBenter** [comment]
Thanks a lot, working perfectly!
Sorry for not creating a pull request. I still needed to figure out Golang...

I have added #408 as compensation. :-)
